.npmignore

```diff
@@ -1,3 +1,3 @@
-# npm ignore everything except /src
+# npm ignore everything except /lib
 *
-!src/*
\ No newline at end of file
+!lib/*
```

Comment:
I'd prefer if we explicitly list node_modules to be ignored instead of ignore everything and then allow lib, but actually node_modules is ignored by default anyway along many dot files https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files

Reply:
If there is a "files" entry, .npmignore is not needed at all anymore, right?

Reply:
I'd still prefer having `gowrapper`, `Dockerfile` and maybe `demo` here and add the `files` field for what we really want to ship to package.json